### DISCUSSION TITLE
Codespace suspended upon rebuilding

### DISCUSSION THREAD

**Discussion by eveningkid** [community/community#3844] [Codespaces] [answered]
Hey,

I have been using Codespaces for a while now and today I've got this alert showing:

<img width="539" alt="Screenshot at Apr 24 11-45-59" src="https://user-images.githubusercontent.com/7188494/115956093-cccd9c80-a4f2-11eb-9b82-c6a696448f85.png">

So I did click on "rebuild", but then the codespace kept being "suspended", then "starting", over and over.

Until it finally started and then greeted me with the exact same alert. I have a lot of projects there and hope not to lose access. Is there anything I could do about this?

Thank you :)

**Comment by matthewisabel** [comment]
Thanks for reporting, we're looking into this @eveningkid

**Comment by eljog** [comment, marked as answer]
Hi @eveningkid , from what we see it appears to be a transient failure. Could you please try again?

Please let us know if the issue persists. Thank you for your patience.

**Comment by eveningkid** [comment]
Hey @eljog,

Just tried and everything worked fine!

Thank you so much for helping out.

Have a great day :)
